# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1463423765 25200
#      Mon May 16 11:36:05 2016 -0700
# Node ID 2e0e4eec0d62475ab9d531b14b09d7327e1e8081
# Parent  f7850bd66042084b5e289a63b3e31c6da9c924c7
Document new function audioformats

* NEWS: Mention new function audioformats.
* audio.txi: Add audioformats DOCSTRING to manual.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -72,16 +72,17 @@ Summary of important user-visible change
     more Matlab-compatible than the previous m-file version.
 
  ** Dialog boxes--errordlg, helpdlg, inputdlg, listdlg, msgbox,
     questdlg, and warndlg--now exclusively use Qt for rendering.
     Java based versions have been removed.
 
  ** Other new functions added in 4.2:
 
+      audioformats
       deg2rad
       evalc
       hash
       im2double
       psi
       odeset
       odeget
       ode45
diff --git a/doc/interpreter/audio.txi b/doc/interpreter/audio.txi
--- a/doc/interpreter/audio.txi
+++ b/doc/interpreter/audio.txi
@@ -37,16 +37,18 @@ information about audio files.  Various 
 wav, flac and ogg vorbis.
 
 @DOCSTRING(audioinfo)
 
 @DOCSTRING(audioread)
 
 @DOCSTRING(audiowrite)
 
+@DOCSTRING(audioformats)
+
 @node Audio Device Information
 @section Audio Device Information
 
 @DOCSTRING(audiodevinfo)
 
 @node Audio Player
 @section Audio Player
 
